src/Servers/IIS/src/Microsoft.AspNetCore.Server.IntegrationTesting.IIS/Microsoft.AspNetCore.Server.IntegrationTesting.IIS.csproj

```diff
@@ -12,11 +12,26 @@
     <UseProjectReferences>true</UseProjectReferences>
     <IsPackable>false</IsPackable>
   </PropertyGroup>
+  
+  <Import Project="..\..\build\assets.props" />
 
   <ItemGroup>
     <EmbeddedResource Include="Http.config" />
   </ItemGroup>
 
+  <Target Name="ValidateNativeComponentsBuilt" AfterTargets="Build" Condition="'$(OS)' == 'Windows_NT'">
+    <Error Text="ANCM has not been built. To build ANCM, you must use MSBuild.exe."
+           Condition="!Exists('$(AspNetCoreModuleV1ShimDll)')
+           OR !Exists('$(AspNetCoreModuleV2ShimDll)')
+           OR !Exists('$(AspNetCoreModuleV2OutOfProcessHandlerDll)')" />
+  </Target>
+
+  <ItemGroup Condition="'$(OS)' == 'Windows_NT'">
+    <NativeProjectReference Include="$(MSBuildThisFileDirectory)..\AspNetCoreModuleV1\AspNetCore\AspNetCore.vcxproj" />
```

Comment:
This condition seems wrong. Wouldn't this project fail to build using `dotnet build` even if native assets are already built?